src/pybind/mgr/dashboard/controllers/ceph_users.py

```diff
@@ -135,7 +135,7 @@ def model(user_entity: str):
     cap_container,
 ])
 
-create_form = Form(path='/cluster/user/create',
+create_form = Form(path='admin/user/create',
```

Comment:
this broke the ceph-users creation form
```suggestion
create_form = Form(path='/user/create',
```